cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/dataset/MetadataResultEntry.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.data2.metadata.dataset;
+
+import io.cdap.cdap.api.metadata.MetadataEntity;
+
+/**
+ * Represents a single metadata entry that is the result of a search
```

Comment:
nit: period at the end of javadoc

Reply:
what do you mean - is a period missing? should every javadoc sentence end with a period?

Reply:
`Represents a single metadata entry that is the result of a search.`

Reply:
ok thank you